ansible/roles/software/ckan/common/vars/bionic.yml

```diff
@@ -2,7 +2,6 @@
 ckan_os_packages:
   - git
   - libbz2-dev
-  - libgdbm-dev
```

Comment:
I think you still want to install this in both bionic and trusty, since both will be hosting the new code (eventually).